test/predicates.jl

```diff
@@ -130,6 +130,14 @@
   end
 
   @testset "intersects" begin
+    function assertintersecsundertransforms(g1, gt, gf)
```

Comment:
Typo, should be `assertintersectsundertransforms` ... or something less verbose entirely

Reply:
The idea behind this is that you provide a geometry `g1`, and one each of another kind of geometry, `gt` and `gf`, where `gt` returns true from `intersects`, and `gf` returns false. 

Do you think this should be type annotated for specifically `Dim = 2`, and `gt` and  `gf` being the same Type?

Reply:
I guess I would want a similar but distinct implementation for the `Dim == 3` case ...

Reply:
Oh, and I suppose that `rotations` `translations` and `transforms` could be defined in their own functions to let them be reused between the 2 and 3 dimension cases.